plugins/inventory/azure_rm.py

```diff
@@ -584,6 +609,66 @@ def _legacy_script_compatible_group_sanitization(name):
 # VMSS VMs (all SS, N specific SS, N resource groups?): SS -> VM -> InstanceView, N NICs, N PublicIPAddress)
 
 
+class ArcHost(object):
+    def __init__(self, arc_model, inventory_client, legacy_name=False):
+        self._inventory_client = inventory_client
+        self._arc_model = arc_model
+        self._instanceview = self._arc_model
+        self._status = self._arc_model['properties'].get('status', {}).lower() # 'Connected'
```

Comment:
at least two spaces before inline comment
```suggestion
        self._status = self._arc_model['properties'].get('status', {}).lower()  # 'Connected'
```